hvpy/tests/test_get_jp2_image.py

```diff
@@ -0,0 +1,19 @@
+from datetime import datetime
+
+from hvpy.api_groups.jpeg2000.get_jp2_image import getJP2ImageInputParameters
+from hvpy.core import execute_api_call
+from hvpy.io import BASE_URL
+
+URL = BASE_URL + "getJP2Image/"
+
+
+def test_get_jp2_image():
+
+    DATE = datetime(2022, 1, 1, 23, 59, 59)
+
+    input = {"date": DATE, "sourceId": 14}
+    input = getJP2ImageInputParameters(**input)
+
+    r = execute_api_call(url=URL, input_parameters=input)
+
+    assert isinstance(r, str)
```

Comment:
Today I learned my API still returns status 200 even on failure... I'm going to update this on my end, but for now look into this failure

It looks like the date isn't being set correctly, here's url that gets sent by your test, according to the API your date looks like this `2022-01-01+23:59:59` but it needs to look like this `2022-01-01T23:59:59Z`

Reply:
Fixed